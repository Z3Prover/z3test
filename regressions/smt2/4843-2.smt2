(define-sort FPN () (_ FloatingPoint 5 11))
(declare-fun x () FPN)
(declare-fun r () FPN)
(assert (= r (fp.roundToIntegral roundNearestTiesToEven x)))
(assert (= r (fp (_ bv0 1) #b10000 (_ bv0 10))))
(check-sat-using smt)
(get-value (x r))
