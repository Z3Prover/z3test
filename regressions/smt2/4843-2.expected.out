sat
((x (fp #b0 #b01111 #b1000000100))
 (r (fp #b0 #b10000 #b0000000000)))
